nixos/modules/services/misc/plex.nix

```diff
@@ -30,6 +30,30 @@ in
         default = "plex";
         description = "Group under which Plex runs.";
       };
+
+
+      managePlugins = mkOption {
```

Comment:
The issue with that is that if `extraPlugins` is empty, we don't know whether the user wants plugin management disabled or just doesn't want any plugins installed. In the latter case, checking for an empty list could leave behind symlinks, causing some plugins not to be uninstalled.